src/cmd/completion.go

```diff
@@ -0,0 +1,209 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/containers/toolbox/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+var completionCmd = &cobra.Command{
+	Use:   "completion [bash|zsh|fish|powershell]",
+	Short: "Generate completion script",
+	Long: `To load completions:
+
+Bash:
+
+  $ source <(toolbox completion bash)
+
+  # To load completions for each session, execute once:
+  # Linux:
+  $ toolbox completion bash > /etc/bash_completion.d/toolbox
+  # macOS:
```

Comment:
Toolbx is Linux-only.  :)

Reply:
Resolved in https://github.com/containers/toolbox/pull/1055 (and https://github.com/containers/toolbox/pull/1113)